WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsForm.swift

```diff
@@ -0,0 +1,166 @@
+import SwiftUI
+import WooFoundation
+
+struct WooShippingCustomsForm: View {
+    @Environment(\.presentationMode) var presentationMode
+    @ObservedObject var viewModel: WooShippingCustomsFormViewModel
+    @State private var isShowingITNInfoWebView = false
+
+    private var contentTypeSelectionView: some View {
+        Menu {
+            // show selection
+            ForEach(WooShippingContentType.allCases, id: \.self) { option in
+                Button {
+                } label: {
```

Comment:
Button action for this Menu and Menu below will be done separately?

Reply:
I suppose you plan to just change those `let`s in `WooShippingCustomsFormViewModel` to `var` and have it like this:
`viewModel.contentType = option`?

Reply:
Exactly, I will do that when implementing the Business Logic.